browser-test/yarn.lock

```diff
@@ -493,13 +493,27 @@
   resolved "https://registry.yarnpkg.com/@bcoe/v8-coverage/-/v8-coverage-0.2.3.tgz#75a2e8b51cb758a7553d6804a5932d7aace75c39"
   integrity sha512-0hYQ8SB4Db5zvZB4axdMHGwEaQjkZzFjQiN9LVYvIFB2nSUHW9tYpxWriPrWDASIxiaXax83REcLxuSdnGPZtw==
 
+"@colors/colors@1.5.0":
```

Comment:
Why were there changes to this file? As I understand, the J2HTML change should be purely a Java change.

Reply:
Isn't yarn.lock similar to package-lock.json? I didn't modify any of the packages in `browser-test/package.json` so these kinds of changes in yarn.lock would probably happen every time someone sets up their browser tests no?